addon/helpers/cat-color-scale.js

```diff
@@ -1,39 +1,35 @@
 import Ember from 'ember';
 import {
-  scaleCategory10,
-  scaleCategory20b,
-  scaleCategory20c,
-  scaleCategory20
+  scaleOrdinal,
+  schemeCategory10,
+  schemeCategory20b,
+  schemeCategory20c,
+  schemeCategory20
 } from 'd3-scale';
 import guidDomainScale from '../utils/guid-domain-scale';
+
 const {
-  get,
   assert,
-  isEmpty
+  isPresent,
 } = Ember;
-const SCALES = {
-  scaleCategory10,
-  scaleCategory20b,
-  scaleCategory20c,
-  scaleCategory20
-};
 
-const WHITELIST = {
-  '10': true,
-  '20': true,
-  '20b': true,
-  '20c': true
+const SCALES = {
+  10: schemeCategory10,
+  '20b': schemeCategory20b,
+  '20c': schemeCategory20c,
+  20: schemeCategory20,
 };
 
 export function catColorScale([type, domain]) {
   let capType = type.toString().toLowerCase();
-  assert(`${type} is not a valid sequential color scale name`, capType in WHITELIST);
 
-  let catScale = get(SCALES, `scaleCategory${capType}`);
-  let scale = guidDomainScale(catScale());
+  let catScale = SCALES[capType];
+  assert(`${type} is not a valid sequential color scale name`, !!catScale);
+
+  let scale = guidDomainScale(catScale);
```

Comment:
Ahh — Given what appears to have changed [here](https://github.com/d3/d3-scale#schemeCategory10), it looks like we'd need to do:

```js
let scale = guidDomainScale(scaleOrdinal(catScale));
```

...though, `catScale` should probably be renamed to `catScheme` to better indicate what it is. (Back when it was `scaleCategory10`, it was actually a scale. `scemeCategory10` looks like it's just an array of colors).